### PR TITLE
feat(example): add google maps screen

### PR DESCRIPTION
**Pull request by ahmedAlaaInstabug**
## Description of the change

1. add Google Maps screen

| Google Maps Screen  | 
|-----------|
|Simulator Screenshot - iPhone 15 Pro - 2024-02-22 at 17 56 24|

- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

Jira ID = MOB-13923

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests

### Code review

- [ ] This pull request has a descriptive title and information useful to a reviewer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

**Comment by mnoseir** [comment]
### Coverage Report
| Label | Coverage | Status |
| ----- | -------- | ------ |
| JavaScript | 96.3% | ✅ |
| Android | 47.5% | ✅ |
| iOS | 41.3% | ✅ |
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 544d3d3a393481868754623ee4eef588fbfd16c2
</p>

File: examples/default/ios/Podfile.lock [hunk 1]
```diff
@@ -793,6 +843,6 @@ SPEC CHECKSUMS:
   Yoga: 8796b55dba14d7004f980b54bcc9833ee45b28ce
   YogaKit: f782866e155069a2cca2517aafea43200b01fd5a
 
-PODFILE CHECKSUM: d169b4508f413ff5d69cdf38428960e0828e6282
+PODFILE CHECKSUM: ad82a989387dcdb1ce80667f16b998a2644cf0b2
 
-COCOAPODS: 1.12.0
+COCOAPODS: 1.14.3
```
Comment (abdelhamid-f-nasser) [inline examples/default/ios/Podfile.lock:848]: Let's revert this as we did not upgrade to this version yet.

**Review by abdelhamid-f-nasser — CHANGES_REQUESTED** [review]
Awesome job as always @ahmedAlaaInstabug, Please check the review 🙏.

File: examples/default/ios/Podfile.lock (hunk 1 above)
Comment (ahmedAlaaInstabug) [reply examples/default/ios/Podfile.lock]: fixed

**Review by abdelhamid-f-nasser — APPROVED** [review]

**Pull request merged**